scripts/utils/recommendation_system.py

```diff
@@ -22,12 +22,9 @@ def build_system(self, x_cols:list):
 
         max_kernels = 30
         sse = get_sum_of_square_errors(features, max_kernels, kmeans_kwargs)
-        draw_graph(x_range=range(1, max_kernels+1), y=sse, 
-                    labels=['Number of Clusters','SSE'])
         
         n_clusters = get_clusters_count(sse, max_kernels)
-        kmeans_kwargs['n_clusters'] = n_clusters
-        self.__model = get_clusters(features, kmeans_kwargs)
+        self.__model = get_clusters(features, n_clusters, kmeans_kwargs)
```

Comment:
you should pass it like
`self.__model = get_clusters(features, n_clusters=n_clusters, **kmeans_kwargs)`, then you would get the passed `n_clusters` value in `**kmeans_kwargs` parameter.